contrib/devtools/README.md

```diff
@@ -40,6 +40,12 @@ would be changed to:
 
 ```// Copyright (c) 2009-2015 The Bitcoin Core developers```
 
+gen-manpages.sh
+===============
+
+A small script to automatically create manpages in ../../doc/man by running the release binaries with the -help option.
+This requires help2man which can be found at: https://www.gnu.org/software/help2man/
```

Comment:
> Why not use a makefile target instead? All of that info would be readily available there.

Do you mean something like [this](https://github.com/bitcoin/bitcoin/blob/master/contrib/devtools/split-debug.sh.in) where it fills in some stuff in the script, or do you mean integrating help2man into the make buildprocess?

Because you can do this: https://www.gnu.org/software/help2man/#Makefile-usage 
until 5 minutes ago I assumed everyone building from source then needs help2man installed,
but this sentence suggest otherwise:
>This usage allows a manual page to be generated by the maintainer and included in the distribution without requiring the end-user to have help2man installed.

Now I am a bit confused :) Guess this needs more research.